minigotchi-ESP32/display.cpp

```diff
@@ -3,58 +3,36 @@
 */
 
 #include "display.h"
+#include <TFT_eSPI.h>
+TFT_eSPI tft; // Define TFT_eSPI object
+
 
-Adafruit_SSD1306* adafruit_display;
```

Comment:
Problem is if you remove all of that we remove all of the display code for other screens when I merge, can we add that back? 

You can probably make a conditional of some sort to enable support for it if selected as a screen


```cpp
if (Config::screen == "CYD") {
    doDisplayStuff();
}
```

Only reason I'm asking for this to be done is because it will maintain consistency with other screens, the code in question will only run if the string `Config::screen` matches one of those.